libs/core/src/lib/notification/notification-group/notification-group.component.ts

```diff
@@ -30,6 +30,10 @@ export class NotificationGroupComponent implements OnChanges, OnInit, CssClassBu
     @Input()
     width: string;
 
+    /** @hidden */
+    @HostBinding('style.height')
+    _height = '100%';
```

Comment:
Why as a hostbinding and not just styled in the css (in fundamental-styles)?

Reply:
moved https://github.com/SAP/fundamental-styles/pull/3783